packages/vscode-extension/WHATISNEW.md

```diff
@@ -6,6 +7,18 @@
 - [2.8.0 - Oct 18 2021](#280---oct-18-2021)
 - [2.7.0 - Sep 17 2021](#270---sep-17-2021)
 
+## 3.1.2 - Jan 10, 2022
+Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+
+New Features:
+- Use Service Principle to login Azure account in CICD template.
+- Support build React Tab app for multiple environments.
+
+Enhancement:
+- Improve the debugging experience on WSL.
+- Support .NET SDK 6.0.
+- Speed up NPM installation.
```

Comment:
Also enhanced the experience of manifest, do we need to mention that?

Reply:
which feature or user story are you talking about? could you paste a link here?

Reply:
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/10920944

Reply:
Thank you, I will add this.